### PR TITLE
Don't generate UTF8 escaping "\u041C\u043E\u0441\u043A\u0432\u0430" in tests 

### PR DESCRIPTION
**Pull request by dstepanov**
_More Unicode fixes..._

Solution looks weird but it's the best I have found. 
http://stackoverflow.com/questions/9806246/switch-off-utf-escaping-in-jsonbuilder

### CODE REVIEW

**Comment by marcingrzejszczak** [comment]
Maybe before merging it first build a local version of accurest and run your build against it?

**Comment by dstepanov** [comment]
Works for me.
I have fixed more bugs https://github.com/Codearte/accurest/pull/161

**Comment by dstepanov** [comment]
@marcingrzejszczak What about this one?

File: accurest-core/src/main/groovy/io/codearte/accurest/builder/SpockMethodBodyBuilder.groovy [hunk 1]
```diff
@@ -117,7 +119,13 @@ abstract class SpockMethodBodyBuilder {
...
 	protected String getBodyAsString() {
 		Object bodyValue = extractServerValueFromBody(request.body.serverValue)
-		return trimRepeatedQuotes(new JsonOutput().toJson(bodyValue))
+		String json = new JsonOutput().toJson(bodyValue)
+		json = convertUnicodeEscapes(json)
+		return trimRepeatedQuotes(json)
+	}
+
+	protected String convertUnicodeEscapes(String json) {
+		StringEscapeUtils.unescapeJavaScript(json)
```
Comment (marcingrzejszczak) [inline accurest-core/src/main/groovy/io/codearte/accurest/builder/SpockMethodBodyBuilder.groovy]: `return`

**Comment by dstepanov** [comment]
@marcingrzejszczak Done

**Pull request merged**